soco/alarms.py

```diff
@@ -16,7 +17,8 @@
 
 
 def is_valid_recurrence(text):
-    """Check that text is a valid recurrence string.
+    """
```

Comment:
Ah, I had not seen from the proposal that this was part of it. Skipping the first line, don't think it is pretty, but not a complaint worth discussing further. I'm just happy to see the doc strings cleaned up.